frontend/tests/components/addMemberModal.ts

```diff
@@ -14,6 +15,10 @@ export class AddMemberModal extends BaseComponent {
     return this.componentLocator.getByLabel(ROLE_LABEL, {exact: true});
   }
 
+  get checkBox(): Locator {
```

Comment:
I'd like this to be called `inviteCheckbox` (no capital B as we treat "checkbox" as a single word) so that it's clear which checkbox is referred to.